### ISSUE TITLE
Tecnical question about stringtie2

### ISSUE THREAD

**Issue by RogerioRibeiro201505534** [gpertea/stringtie#315]
Good morning. 
I've been tasked with the assembly of a transcriptome for my species of interests, using the genome as a guide and 23 RNA-seq samples

After my initial assembly,  I noticed that quite a few transcripts In my assembly covered two or more reference genes. After manual inspection, I also noticed that a great deal of these cases actually has only a few reads supporting the splicing sites. This is a known problem with stringtie. to solve this. My supervisor suggested that I concatenate all the alignments from all the 23 samples, and then feed that file to stringtie, with the -j option. I've since read the original paper on stringtie and got the idea that transcript expression levels are important for the assembly, but I'm not to sure not this.

Is it correct to use this approach, or I should assemble each sample individually?

**Comment by mpertea** [comment]
Yes, your assumption is correct: the assemblies will be more accurate if the samples are assembled individually. Better to impose a -j threshold on each sample individually.

**Comment by RogerioRibeiro201505534** [comment]
Thanks for your answer! 
As a follow up, do you have any suggestion for merge data from Illumina and Nanopore? I've been using the stringtie --merge setting the isoform fraction to 0
